tests/default_vars.sh

```diff
@@ -624,6 +624,8 @@ export DO_UGWP_V1_W_GSLDRAG=.false.
 export DO_UGWP_V0_OROG_ONLY=.false.
 export DO_GSL_DRAG_LS_BL=.false.
 export DO_GSL_DRAG_SS=.true.
+export DO_GWD_OPT_PSL=.true.
```

Comment:
@jkbk2004 Are these meant to be defaults for all tests, or only the gfsv17 tests?

Reply:
I think most tests see: https://github.com/ufs-community/ufs-weather-model/pull/2290#issuecomment-2189195606 and https://github.com/ufs-community/ufs-weather-model/pull/2290#issuecomment-2189268134

Reply:
I prefer to move those two parameters at https://github.com/Qingfu-Liu/ufs-weather-model/tree/HR4-GWD-update/tests/tests level but parm/global_control.nml.IN is used for quite some tests. Let me check if I can update. @JessicaMeixner-NOAA commit schedule tomorrow and plan to merge by Friday, ok?

Reply:
We need to set  DO_GWD_OPT_PSL=.false.  in default_vars.sh since it is used by all RTs.   In RTs related to GFSv17 and GEFSv13 tests reset:  

DO_GWD_OPT_PSL=.true.    
DO_GSL_DRAG_SS=.false.
DO_GSL_DRAG_TOFD=.true.

Note that regional models and the global model with nest (AR test?) should not use the new GWD settings.  The corresponding GWD datasets for these applications were not created.   @JongilHan66 please help check the RT test settings and confirm

Reply:
@jkbk2004 that's fine, thanks for all your help with this.

Reply:
I think the confusion comes from the different approaches for setting parameters between global workflow and the RTs.   We need to double check to make sure they are set correctly in both.

Reply:
I can run a test from the global-workflow PR so we can double check those settings.  Is the C48mx500 test okay or do we need the higher resolution?

Reply:
@JessicaMeixner-NOAA  I cannot log on WCOSS2 so I do not know if the new C48 GWD dataset is available.  I asked to create the new datasets for all global resolutions.   C48mx500 test should suffice.

Reply:
@yangfanglin data sets were created for all resolutions from what I can tell... I probably will run this test on hera.

Reply:
@yangfanglin Can I ask what tests would be considered 'gefsv13'? I'm not sure we have an exact analog for that yet, since it will be s2swa but waves on the structured mesh, is that right? We only have the p8 version of that configuration.

Reply:
@DeniseWorthen I think s2swa is the one  (@bingfu-NOAA )